w2/num.py

```diff
@@ -25,15 +25,17 @@ def mid(self):
     def div(self):
         return 0 if self.n < 2 else (self.m2 / (self.n - 1)) ** 0.5
 
-    def small(self):
-        return the["cohen"] * self.div()
-
     def norm(self, x):
         return x if x == "?" else (x - self.lo) / (self.hi - self.lo + 1E-30)
 
+    """
+    # Currently not supported
+    def small(self):
+        return the["cohen"] * self.div()
+
     def like(self, x, _):
```

Comment:
Should the Like be implemented? Prof said no right?

Reply:
We don't need to implement it for this weeks assignment. So I just comment it out.